packages/FormElement/src/FormElement.js

```diff
@@ -71,85 +72,92 @@ function FormElement(props) {
+      return React.cloneElement(extractedChildren["FormElement.Error"], {
+        ariaErrorId,
+      });
     }
 
-    if (extratedChildren["FormElement.Description"]) {
-      return React.cloneElement(extratedChildren["FormElement.Description"], {
+    if (extractedChildren["FormElement.Description"]) {
+      return React.cloneElement(extractedChildren["FormElement.Description"], {
         ariaDescriptionId,
       });
     }
 
     return null;
   }
 
+  function renderInstructions() {
+    if (extractedChildren["FormElement.Instructions"]) {
+      return React.cloneElement(extractedChildren["FormElement.Instructions"], {
+        ariaInstructionsId,
+      });
+    }
+    return null;
+  }
+
+  const nativeChildTypes = ["input", "textarea", "select"];
+
+  const renderFormElementChild = child => (
+    <div data-pka-anchor="form-element.child" key={`${child.key}`} css={formElementChildStyle}>
+      {child}
+    </div>
+  );
+
+  const ariaDescribedByIds = `${ariaErrorId} ${ariaInstructionsId} ${ariaDescriptionId}`;
+
+  const childExtendedProps = {
+    "aria-describedby": ariaDescribedByIds,
+    id: inputId,
+  };
+
   return (
     <div css={formElementStyles} isInline={isInline} size={size} isDisabled={isDisabled} {...moreProps}>
       <Label
         hasOptionalLabel={hasOptionalLabel}
         hasRequiredLabel={hasRequiredLabel}
-        help={extratedChildren["FormElement.Help"]}
+        help={extractedChildren["FormElement.Help"]}
         id={inputId}
         isInline={isInline}
         isVisuallyHidden={isLabelVisuallyHidden}
         label={label}
       />
-
-      {extratedChildren.children.map(child => {
-        if (React.isValidElement(child)) {
-          const extendedProps = isString(child.type)
-            ? {
-                "aria-describedby": ariaDescriptionId,
-                id: inputId,
-                disabled: isDisabled,
-                readOnly: isReadOnly,
-              }
-            : {
-                "aria-describedby": ariaDescriptionId,
-                hasError,
-                id: inputId,
-                isDisabled,
-                isReadOnly,
-                size,
-              };
-          const clonedChild = React.cloneElement(child, extendedProps);
-
-          return (
-            <React.Fragment key={child.key}>
-              {isInline ? (
-                <div css={inlineContainerStyles}>
-                  {clonedChild}
-                  {renderFooter()}
-                </div>
-              ) : (
-                <React.Fragment>
-                  {clonedChild}
-                  {renderFooter()}
-                </React.Fragment>
-              )}
-            </React.Fragment>
-          );
-        }
-
-        return child;
-      })}
+      <div css={isInline ? inlineContainerStyles : null}>
+        {renderInstructions()}
+        {extractedChildren.children.map(child => {
+          if (isString(child.type) && nativeChildTypes.includes(child.type)) {
```

Comment:
I think it would read better to extract the `isString(child.type)` into a function that also does the `nativeChildTypes.includes(child.type)`.